rpc/core/tx.go

```diff
@@ -100,5 +102,24 @@ func TxSearch(ctx *rpctypes.Context, query string, prove bool, page, perPage int
 		}
 	}
 
+	if len(apiResults) > 1 {
+		switch orderBy {
+		case "desc":
+			sort.Slice(apiResults, func(i, j int) bool {
+				if apiResults[i].Height == apiResults[j].Height {
+					return apiResults[i].Index > apiResults[j].Index
+				}
+				return apiResults[i].Height > apiResults[j].Height
+			})
+		default: // asc
```

Comment:
I think this should explicitly handle `"asc"` and `""`, otherwise return error. As it is, I believe e.g. `"DESC"` would silently be interpreted as `"asc"´.

Reply:
Great catch!